### DISCUSSION TITLE
Could use instructions on setting up development environment

### DISCUSSION THREAD

**Discussion by jocassid** [CycloneDX/cyclonedx-python#408] [Q&A] [answered]
How do I set up cyclonedx-python in poetry to use my local copy of cyclonedx-python-lib instead of the version that the poetry.lock file will pull in?    I'd like to be able to run the cyclonedx-bom script via the debugger and follow code execution into cyclonedx-python-lib.  Some instructions on how to do this would be appreciated.

**Comment by jkowalleck** [comment, marked as answer]
change the affected dependency to a local one, as described here: https://python-poetry.org/docs/dependency-specification/#path-dependencies
